opencti-platform/opencti-front/src/private/components/common/stix_domain_objects/StixDomainObjectHeader.jsx

```diff
@@ -240,6 +240,15 @@ const StixDomainObjectHeader = (props) => {
 
   const onSubmitCreateAlias = (values, { resetForm, setSubmitting }) => {
     const currentAliases = getCurrentAliases();
+    const normalizeNameEntity = (stixDomainObject.name).toLowerCase().trim();
+    const normalizeNewAlias = newAlias.toLowerCase().trim();
+    if (normalizeNameEntity === normalizeNewAlias) {
+      setOpenAlias(false);
+      setOpenCommitCreate(false);
+      setNewAlias('');
+      resetForm();
+      return;
```

Comment:
I suggest we display an error message here to notify the user that the alias can't be named the same as the entity name.